### ISSUE TITLE
Extra / in hreflang links in the <head>

### ISSUE THREAD

**Issue by hamidpeywasti** [terminal42/contao-changelanguage#252]
Hi,
There are an Extra "/" in **href links**.

This is an example from contao.org:

```
<link rel="alternate" hreflang="x-default" href="https://contao.org//de/media">
<link rel="alternate" hreflang="en" href="https://contao.org//en/media">
<link rel="alternate" hreflang="es" href="https://contao.org//es/media">
<link rel="alternate" hreflang="fr" href="https://contao.org//fr/media">
```

![image](https://github.com/user-attachments/assets/f30f69c4-5908-4c98-86a3-c99f7a1eb779)

**Comment by aschempp** [comment]
any idea where this is coming from?

**Comment by fritzmg** [comment]
Possibly here:

https://github.com/terminal42/contao-changelanguage/blob/867518e91460d5d6ed6baf25008644bcb2182a5a/src/Helper/AlternateLinks.php#L82

`Environment::get('base')` will contain a trailing slash and the given `$href` might be path absolute (default in Contao 5).

**Comment by aschempp** [comment]
should be fixed in eacbb98a4241e13717dd4f94ce7284c96e38395b
